src/test/java/com/redhat/devtools/lsp4ij/features/formatting/TypeScriptEditorImprovementsTest.java

```diff
@@ -545,4 +546,70 @@ export class Foo {
     private static String adjustIndent(@NotNull String fileBody, boolean useTabCharacter) {
         return useTabCharacter ? fileBody.replace("    ", "\t") : fileBody;
     }
+
+    // These tests exercise the behavior of LSPImprovedTextMateNestedBracesTypedHandler
+
+    public void testImprovedTextMateNestedBraces() {
```

Comment:
Confirms that the new TextMate-specific nested braces improvements are _enabled by default_ for user-defined language servers.